arkouda/strings.py

```diff
@@ -525,6 +525,55 @@ def is_title(self) -> pdarray:
             generic_msg(cmd="checkChars", args=f"isTitle {self.objtype} {self.entry.name}")
         )
 
+    @typechecked
+    def strip(self, chars: Optional[Union[bytes, str_scalars]] = "") -> Strings:
+        """
+        Returns a new Strings object with all leading and trailing characters removed. The chars
```

Comment:
I know this based on the `str.strip` doc strings, but i think we should be explicit that it's the characters in `chars`

`Returns a new Strings object with all leading and trailing occurrences of characters contained in chars removed`